.github/workflows/firesim-run-tests.yml

```diff
@@ -82,17 +82,20 @@ jobs:
         run: ./.github/scripts/launch-manager-instance.py
       - name: Setup N Github Actions Runners on AWS instance
         run: ./.github/scripts/setup-manager-self-hosted.py
-
```

Comment:
Hmmm. I'm slightly concerned about combining the GH-A runner init and the other init into one area but if CI passes then I'm fine with it.